Automattic/wp-calypso
# Disabling monthly billing on Premium and Personal plans

#### Changes proposed in this Pull Request

* The PR implements a test for disabling the monthly option for Premium and Personal plans on the plans page in the signup flow.
* Context for this test is available at p58i-bkW-p2.
* I'll update this PR with a link to the experiment post when it's up.

Here's a screenshot of the design provided by @lucasmdo:

<img width="1035" alt="disable-monthly-test" src="https://user-images.githubusercontent.com/35781181/138381933-6ba75f21-9ab0-49ea-ab5c-8ca1c41fc520.png">

#### Testing instructions



* Checkout this PR and start Calypso.
* Add yourself to the test variant for the Explat test named `disabled_monthly_personal_premium`.
* Navigate to the `/start/new` flow and continue to the Plans step.
* Click the Monthly tab and confirm that Personal and Premium are not selectable.
* Click back to the Annual tab and confirm that they are selectable.
* Add yourself to the control variant and confirm that it still works as expected.



Related to #
